examples/relay-angular-chat/src/app/app.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component } from '@angular/core';
+import { Waku, WakuMessage } from 'js-waku';
+import protons from "protons";
+
+const proto = protons(`
+message SimpleChatMessage {
+  uint64 timestamp = 1;
+  string text = 2;
+}
+`);
+
+interface MessageInterface {
+  timestamp: Date,
+  text: string
+}
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+
+export class AppComponent {
+  
+  contentTopic: string = `/relay-angular-chat/1/chat/proto`;
+  messages: MessageInterface[] = [];
+  messageCount: number = 0;
+  title: string = 'relay-angular-chat';
+  waku!: Waku;
+  wakuStatus: string = 'None';
+
+  ngOnInit(): void {
```

Comment:
In this basic app, it's when the whole app starts as the 'app' is the component. 

In a 'real' angular app where there are multiple services and components to connect to waku, fetch and display the messages etc. The waku service would be a singleton and be injected (created) when needed in a component and destroyed when not, e.g. a chat window on the page. Or not. It would be up to the dApp developer.

Reply:
ok thanks